pull_request_template.md

```diff
@@ -1,7 +1,11 @@
+## Reminders
+1. Did I review my own code.
+2. Is there only 1 commit.
+
 ## Checklist before requesting a review
 - [ ] I have performed a self-review of my code
 - [ ] I have added thorough tests. (If tests are not applicable, explain why in the next section.)
 
```

Comment:
The reminders can go under this subheading as a checkbox ->
- [ ] Did I squash my commits into 1 commit?